### ISSUE TITLE
Multiple sura for specific sources

### ISSUE THREAD

**Issue by obadx** [obadx/prepare-quran-dataset#28]
while giving the same key to `specific_sources` It does not complain. Example
```json
{
    "001": "https://cdns1.zekr.online/quran/10341/1/128.mp3",
    "001": "https://cdns1.zekr.online/quran/10341/110/128.mp3",
    "001": "https://cdns1.zekr.online/quran/10341/111/128.mp3"
}
```
It should raise duplicate key Error

**Comment by obadx** [comment]
We will move to yaml style إن شاء الله

**Comment by obadx** [comment]
Solved by: #32
